test_runner/batch_others/test_proxy.py

```diff
@@ -83,16 +83,21 @@ def create_and_send_db_inf(local_vanilla_pg, psql_session_id):
     host = local_vanilla_pg.default_options['host']
     dbname = local_vanilla_pg.default_options['dbname']
 
+    db_info =  \
+        '{"session_id": "' + str(psql_session_id) + '"' + ',' + \
+        '"result":{''"Success":{ ' + \
+        '"host":"' + host + '","port":' + str(port) + ',' + \
+        '"dbname":"' + dbname + '","user":"' + pg_user + '",' + \
+        '"password":"' + pg_password + '"}}}'
```

Comment:
does `json.dumps(db_info_dict)` do the job?

Reply:
yes